llama.h

```diff
@@ -437,6 +437,20 @@ extern "C" {
                       const char * path_base_model,
                          int32_t   n_threads);
 
+    // Apply a loaded control vector to a llama_context, or if data is NULL, clear
+    // the currently loaded vector.
+    // n_embd should be the size of a single layer's control, and data should point
+    // to an n_embd x n_layers buffer starting from layer 1.
+    // il_start and il_end are the layer range the vector should apply to (both inclusive)
+    // See llama_control_vector_load in common to load a control vector.
+    LLAMA_API int32_t llama_control_vector_apply(
+            struct llama_context * lctx,
+                           float * data,
+                          size_t   len,
+                             int   n_embd,
```

Comment:
You can get `n_embd` from `lctx->model.hparams.n_embd`

Reply:
Yes, but I want to track the n_embd of the vector to make sure that the user doesn't try to load a vector with a differing n_embd than the model--the n_embd in this definition is the n_embd of the vector, not the model.